src/EditorFeatures/Core/Implementation/KeywordHighlighting/HighlighterViewTaggerProvider.cs

```diff
@@ -53,12 +53,14 @@ internal class HighlighterViewTaggerProvider : AsynchronousViewTaggerProvider<Ke
             _highlightingService = highlightingService;
         }
 
+        protected override TaggerDelay EventChangeDelay => TaggerDelay.NearImmediate;
+
         protected override ITaggerEventSource CreateEventSource(ITextView textView, ITextBuffer subjectBuffer)
         {
             return TaggerEventSources.Compose(
-                TaggerEventSources.OnTextChanged(subjectBuffer, TaggerDelay.OnIdle),
```

Comment:
note: being OnIdle here wasn't doing anything.  any time the text changed, we'd get teh caret position changing.  so this was always near-immediate anyways =-o